development/compiling/compiling_for_android.rst

```diff
@@ -199,6 +199,30 @@ file in the ``bin\`` directory of your Godot source folder, so that the
 next time you build you will automatically have the custom templates
 referenced.
 
+.. _doc_compiling_for_android_creating_source_zip:
+
+Creating an Android source ZIP
+------------------------------
+
+To create an Android source ZIP (which is used for custom builds), run the
+following command at the root of the Godot source repository folder::
+
+    cd platform/android/java
+    # On Windows
+    .\gradlew zipCustomBuild
+    # On Linux and macOS
+    ./gradlew zipCustomBuild
+
+You can run this task before or after compiling export templates, as the
+generated ZIP does not contain compiled binaries. Once the ``zipCustomBuild``
+task has finished running, ``android_source.zip`` will be generated in the
+``bin/`` folder.
```

Comment:
@Calinou the `gradlew generateGodotTemplates` command also generates the `android_source.zip` file.
Is this step solely to cover the scenario that only the `android_source.zip` file is needed?

Reply:
> gradlew generateGodotTemplates command also generates the android_source.zip file.

This is important information that should be captured in the documentation.

Reply:
This statement:

> You can run this task before or after compiling export templates, as the generated ZIP does not contain compiled binaries.

Must be changed to 

> You can must run this task after compiling the export templates, as the generated android_source.zip contains the compiled binaries.